### PR TITLE
Contact Form Mobile version Template 1 (Pair programming)

### PR DESCRIPTION
**Pull request by kessio**
This pull request contains a created contact form parsed from Figma Template 1 UI. The contact form was done in collaboration with two other authors who have been listed on the README file of this pull. The objective of this project is to work on linting errors, pair programming, form creation, form validation, and parsing UI from Figma.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -378,4 +378,93 @@ a.my-lang {
   font-size: 20px;
 }
 
+/* contact section */
+.contact-section {
```
Comment (blessedjasonmwanza) [inline style.css:382]: - [ ] The contact section should be centered and cover the entire width of a mobile device as per [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Kindly consider centering it as you did for the projects section. Here is a UI difference reference to help you with this 🤓 
### Your current UI
image
### Expected output

image

Use chrome dev tools to inspect and simulate how the project can look on different mobile devices. Checkout this [Article](https://www.browserstack.com/guide/view-mobile-version-of-website-on-chrome) for your reference.

File: index.html [hunk 2]
```diff
@@ -8,172 +9,206 @@
...
+                    <input type="email" placeholder="youremail@email.com" id="email" required="required">
+                </div>
+                <div class="form-contact">
+                    <textarea placeholder="Write your message here" rows="4" cols="50" maxlength="500" required="required"></textarea>
+                </div>
+                <div class="form-contact">
+                    <a href="#">
+                        <div class="contact-btn">Get In Touch</div>
+                    </a>
+                </div>
```
Comment (blessedjasonmwanza) [inline index.html:207]: - [ ] The UI looks good 👍🏽 . However, for structural-semantic guidelines and in order for any `form`  to be submitted to the set action URL, It is required to use either a `button` with type `submit` or an `input` field with type `submit`. Here is a nice [article](https://www.w3schools.com/html/html_forms.asp#:~:text=The%20Submit%20Button,in%20the%20form's%20action%20attribute.) to help you with this part.

File: style.css (hunk 1 above)
Comment (blessedjasonmwanza) [inline style.css:382]: - [ ] Kudos 👏🏽 for using `display: flex` in other parts of the project. Kindly note that you're also required to use flex here and throughout your project.

**Review by blessedjasonmwanza — CHANGES_REQUESTED** [review]
# 👀  Status: Changes Required ♻️ 
> ### Hi @kessio && @mj-e-boc  && @PraisesPJMT ,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @blessedjasonmwanza in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css (hunk 1 above)
Comment (blessedjasonmwanza) [reply style.css:398]: Hi @kessio, The Figma width is portrayed on a 375 simulated device. You can utilize the power of CSS media queries and percentages to fill up an entire screen when the user is on smaller screens. Hopefully, this helps.

File: index.html [hunk 3]
```diff
@@ -8,172 +9,205 @@
...
+                    </div>
+                    <div class="form-contact">
+                        <input type="email" placeholder="youremail@email.com" id="email" required="required">
+                    </div>
+                    <div class="form-contact">
+                        <textarea placeholder="Write your message here" rows="4" cols="50" maxlength="500" required="required"></textarea>
+                    </div>
+                    <div class="form-contact">
+                        <button type="submit" class="contact-btn">Get in Touch</button>
+                    </div>
```
Comment (Bondok6) [inline index.html:207]: - [OPTIONAL] According to the requirements, You should Implement the button interactions (enable, hover, etc.). Kindly, add the button interactions based on your Figma Design 👌.



image

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi Team 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great work on making the changes requested by the previous reviewer ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**